tests/handlers/records-read.spec.ts

```diff
@@ -632,7 +633,292 @@ export function testRecordsReadHandler(): void {
         });
 
+        describe('protocolPath based reads', () => {
+          it('should not allow only `protocol` to be set without a `protocolPath` nor `recordId`', async () => {
+            const alice = await DidKeyResolver.generate();
+
+            // insert public data
+            const { message, dataStream } = await TestDataGenerator.generateRecordsWrite({ author: alice, published: true });
+            const writeReply = await dwn.processMessage(alice.did, message, dataStream);
+            expect(writeReply.status.code).to.equal(202);
+
+            // create with protocol and protocolPath to avoid the failure within RecordsRead.create()
+            const recordsRead = await RecordsRead.create({
+              protocol                    : 'example.org/TestProto',
+              protocolPath                : 'proto/path',
+              authorizationSignatureInput : Jws.createSignatureInput(alice)
+            });
+
+            // delete protocolPath leaving only protocol to induce error below
+            delete recordsRead.message.descriptor.protocolPath;
+
+            const readReply = await dwn.handleRecordsRead(alice.did, recordsRead.message);
+            expect(readReply.status.code).to.equal(400);
+            expect(readReply.status.detail).to.contain('must have required property \'recordId\'');
+          });
+
+          it('should not allow only `protocolPath` to be set without a `protocol` nor `recordId`', async () => {
+            const alice = await DidKeyResolver.generate();
+
+            // insert public data
+            const { message, dataStream } = await TestDataGenerator.generateRecordsWrite({ author: alice, published: true });
+            const writeReply = await dwn.processMessage(alice.did, message, dataStream);
+            expect(writeReply.status.code).to.equal(202);
+
+            // create with protocol and protocolPath to avoid the failure within RecordsRead.create()
+            const recordsRead = await RecordsRead.create({
+              protocol                    : 'example.org/TestProto',
+              protocolPath                : 'proto/path',
+              authorizationSignatureInput : Jws.createSignatureInput(alice)
+            });
+
+            // delete protocolPath leaving only protocol to induce error below
+            delete recordsRead.message.descriptor.protocol;
+
+            const readReply = await dwn.handleRecordsRead(alice.did, recordsRead.message);
+            expect(readReply.status.code).to.equal(400);
+            expect(readReply.status.detail).to.contain('must have required property \'recordId\'');
+          });
+
+          it('should retrieve the latest protocol record matching the path', async () => {
+            const testCases = [];
+            const alice = await DidKeyResolver.generate();
+
+            const protocolDefinition = emailProtocolDefinition as ProtocolDefinition;
+
+            // Install social-media protocol on Alice's DWN
```

Comment:
```suggestion
            // Install protocol on Alice's DWN
```
Since this the `email` protocol fixture, not `social-media`